lib/cldr/messages/messages.ex

```diff
@@ -54,7 +54,25 @@ defmodule Cldr.Message do
   @spec format(String.t(), arguments(), options()) ::
           {:ok, String.t()} | {:error, {module(), String.t()}}
 
-  def format(message, args \\ [],options \\ []) when is_binary(message) do
+  def format(message, args \\ [], options \\ []) when is_binary(message) do
```

Comment:
I have flipped `format` / `format!` because the `rescue` loses detailed information of the `KeyError`.

This way `format!` has the full error and `format` only takes the message.